project/main/util_helper.py

```diff
@@ -196,7 +196,6 @@ def validaPollutant(val, sensor_name):
             calificacionInca = 600
         else:
             calificacionInca = -1
-        return calificacionInca
```

Comment:
The build fails but the thing is that build was failing even before I made the PR. I don't think any changes made by  me introduces a bug.

Reply:
We are currently missing a few tests and some are wrong too, that's why they are failing. Your changes will absolutely not introduce a bug, thanks!

Reply:
However, the return_color variable must have 'green' as a default value in case none of the if-elif statements are _True_. Don't worry, I will add it. Thanks again!
![image](https://user-images.githubusercontent.com/71281833/134713893-a10a1e3d-cb0d-4fef-95ae-5373c5dc87ee.png)